components.css

```diff
@@ -4,6 +4,9 @@
   --warning: #fde68a;
   --info: #17a2b8;
   --white: white;
+  --black: black;
```

Comment:
you could have used "color" keyword before color name to make it easy while searching for colors